getting-started/windows-calico/openshift/installation.md

```diff
@@ -103,17 +103,21 @@ openshift-install create cluster
 
 Once the above command is complete, you can verify {{site.prodname}} is installed by verifying the components are available with the following command.
 
-```
+```bash
 oc get tigerastatus
 ```
 
 > **Note**: To get more information, add `-o yaml` to the above command.
 
-Next, install calicoctl and [ensure that strict affinity is true]({{site.baseurl}}/getting-started/windows-calico/quickstart#configure-strict-affinity-for-clusters-using-calico-networking):
+Next, [install calicoctl]({{site.baseurl}}/getting-started/clis/calicoctl/install) and ensure strict affinity is true:
+
+```bash
+calicoctl ipam configure --strictaffinity=true
+```
```

Comment:
@lwr20 this addresses your feedback of not requiring the user go to another page to get this incantation :)